ql/src/java/org/apache/hadoop/hive/ql/security/authorization/StorageBasedAuthorizationProvider.java

```diff
@@ -28,7 +28,7 @@
 import javax.security.auth.login.LoginException;
 
 import org.apache.hadoop.hive.conf.Constants;
```

Comment:
You can remove Constants just like in the previous patch. Same is true for other classes where you've removed unused imports in the previous patch, but not in this one for some reason.

Reply:
Removed